src/main/_docs/setup/setup-configuration.md

```diff
@@ -92,6 +92,8 @@ Refer to [GitLab using OAuth]({{base}}{{site.links["ide-git-svn"]}}#gitlab-oauth
 # Stacks
 [Stacks]({{ base }}{{site.links["devops-runtime-stacks"]}}) define the recipes used to create workspace runtimes. They appear in the stack library of the dashboard. You can create your own.
 
+`CHE_PREDEFINED_STACKS_RELOAD__ON__START` (false by default) defines stack loading policy. When set to false, stacks are loaded from a json file only once - when database is initialized. When set to true, json is sourced every time Che server starts.
+
```

Comment:
So how can I alter the stacks list? How can I get a new stacks provided on a new version of Che without loosing my stacks?

This needs to be explicit in the docs.